File: python/modelcreator/java/model/events/EventData.java
package model.events;

import java.util.HashMap;

public class EventData {
	private String id;
	private String name;
	private String trigger;
	private HashMap<String, String> assignments;
	
	public EventData(String id, String name, String trigger, HashMap<String, String> assignments){
		this.id = id;
		this.name = name;
		this.trigger = trigger;
		this.assignments = assignments;
	}
	
	public String getId() {
		return id;
	}
	public String getName() {
		return name;
	}
	public String getTrigger() {
		return trigger;
	}
	public HashMap<String, String> getAssignments() {
		return assignments;
	}
}

File: python/modelcreator/modelcreator/events/eventdata.py
'''
Creating the simulation event data.
The EventData is used to generate the SBML events for the model.

Created on Jul 2, 2014
@author: mkoenig
'''

class EventData():
    def __init__(self, eid, name, trigger, assignments):
        self.eid = eid
        self.name = name
        self.trigger = trigger
        self.assignments = assignments
    
    def info(self):
        print '-' * 20
        print self.eid
        print self.name
        print self.trigger
        print self.assignments
        